frontend/src/App.tsx

```diff
@@ -16,18 +26,19 @@ class App extends PureComponent<Props> {
 
```

Comment:
_nit/out of scope of this PR_: I'm getting an es-lint error here for a "useless constructor". Do we really need it?